lms/djangoapps/certificates/views/webview.py

```diff
@@ -517,6 +517,13 @@ def render_html_view(request, user_id, course_id):
         log.info(error_str, course_id, user_id, str(exception))
         return _render_invalid_certificate(course_id, platform_name, configuration)
 
+    if not certificates_viewable_for_course(course):
```

Comment:
I think it's fine to keep this logic separate so we can have different log messages.

Reply:
@ahsan-ul-haq yes @bderusha is right, I did not notice log message previously. You are go to go once build pass.